health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -547,7 +569,13 @@ const ViewHierarchy = () => {
                                     />
                                 </div>
                                 <div style={{height:"2rem"}}></div>
-                            </div>                  
+                            </div>
+                            {uiValError && <InfoCard
+                              label="Info"
+                              text={uiErrorMsg}
+                              variant="error"
+                            /> }  
+                            <div style={{marginBottom:"2rem"}}></div>               
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)